enterprise/frontend/src/metabase-enterprise/whitelabel/components/MetabaseLinksToggleDescription.tsx

```diff
@@ -0,0 +1,47 @@
+import { useState } from "react";
+import type { MouseEvent } from "react";
+import { t } from "ttag";
+import { Anchor, Popover, Stack, Text } from "metabase/ui";
+
+export function MetabaseLinksToggleDescription() {
+  const [opened, setOpened] = useState(false);
+  return (
+    <Popover
+      key="popover"
+      position="top-start"
+      opened={opened}
+      onClose={() => setOpened(false)}
+    >
+      <Popover.Target>
+        <Anchor
+          /**
+           * This expands the vertical click area, so it's impossible to click above
+           * or below the element and accidentally toggle the switch.
+           */
+          display="inline-block"
+          style={{ userSelect: "none", cursor: "pointer" }}
+          onClick={(event: MouseEvent) => {
+            event.preventDefault();
```

Comment:
this prevents clicking the Learn more link and flips the toggle which is totally weird.